openedx/core/djangoapps/credentials/tests/test_utils.py

```diff
@@ -128,21 +129,39 @@ def test_get_user_programs_credentials(self):
         """Verify program credentials data can be retrieved and parsed correctly."""
         # create credentials and program configuration
         self.create_credentials_config()
-        self.create_programs_config()
 
         # Mocking the API responses from programs and credentials
-        self.mock_programs_api()
-        self.mock_credentials_api(self.user, reset_url=False)
+        uuidfirst, uuidsecond = str(uuid.uuid4()), str(uuid.uuid4())
+        credentials_api_response = {
+            "next": None,
+            "results": [
+                factories.UserCredential(
+                    id=1, username='test',
```

Comment:
Do you need to specify the id here? Same question for the other spots in here where you specify this.